src/udev/udevd.c

```diff
@@ -293,6 +293,8 @@ static void manager_free(Manager *manager) {
         if (!manager)
                 return;
 
+        manager->monitor = sd_device_monitor_unref(manager->monitor);
```

Comment:
It's not necessary to assign value here.

Reply:
I think it's reasonable to do it. We call other functions which take `manager` as argument below, and while they don't look at `manager->monitor`, somebody might change something in one of them later, and avoiding a stale pointer just makes the whole thing more robust.

Reply:
OK.